### PR TITLE
MM-31518 Cloud Docs Technical Review Updates

### PR DESCRIPTION
**Pull request by cwarnermm**
Docs ticket: https://mattermost.atlassian.net/browse/MM-31518

Updated:
- Cloud Admin Guide > Integrations > Embedding Mattermost in Other Applications
   - Removed the following subsections
      - Embedding Mattermost in Mobile Applications (irrelevant unless you are adding React native to your native app or you build a react native application yourself)
      - Mobile Applications Offering Mattermost as a Web View (classic apps are outdated and these docs show a way to embed the whole web app in a container)
      - Mobile Applications Offering Mattermost with React Native (these mobile app components can't easily be extracted)
   - Removed the Contents links, given that the page sections have been reduced by half

### CODE REVIEW

File: source/cloud/cloud-mobile/troubleshooting-mobile.rst [hunk 1]
```diff
@@ -2,49 +2,15 @@
...
-This can happen if the server running Mattermost has its mime types not set up correctly. A server running Linux has this file located in ``/etc/mime.types``. This might vary depending on your specific OS and distribution.
-
-Some distributions also ship without ``mailcap`` which can result in missing or incorrectly configured mime types.
-
-None of these solve my problem!
+I have a different problem!
 -------------------------------
 
-For more troubleshooting help, `open a new topic in our forums <https://forum.mattermost.org/c/trouble-shoot>`__ with steps to reproduce your issue. If you're an Enterprise Edition subscriber, you may open a support ticket in the `Enterprise Edition Support portal <https://mattermost.zendesk.com/hc/en-us/requests/new>`_.
+For more troubleshooting help, `open a new topic in our forums <https://forum.mattermost.org/c/trouble-shoot>`__ with steps to reproduce your issue. If you're a Cloud Enterprise subscriber, you may open a support ticket in the `Mattermost Support portal <https://mattermost.zendesk.com/hc/en-us/requests/new>`_.
```
Comment (marianunez) [inline source/cloud/cloud-mobile/troubleshooting-mobile.rst:13]: @adamjclarkson For cloud support should we be directing them to: https://customers.mattermost.com/cloud/contact-us ? Not sure if the forum or zendesk makes sense here

**Review by enahum — APPROVED** [review]

File: source/cloud/cloud-integrations/cloud-incoming-webhooks.rst [hunk 2]
```diff
@@ -70,24 +70,24 @@ This content will be displayed in the Town Square channel.
 Messages with advanced formatting can be created by including an :doc:`attachment array <message-attachments>` and :doc:`interactive message buttons <interactive-messages>` in the JSON payload.
 
 .. note::
-  `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames. Enable them from **System Console > Integrations > Custom Integrations** in prior versions or **System Console > Integrations > Integration Management** in versions after 5.12 or ask your System Administrator to do so. If not enabled, the username is set to `webhook`.
+  `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to ``true`` in ``config.json`` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your System Administrator to do so. If not enabled, the username is set to `webhook`.
```
Comment (justinegeffen) [inline source/cloud/cloud-integrations/cloud-incoming-webhooks.rst:73]:
```suggestion
  `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to ``true`` in ``config.json`` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your System Admin to do so. If not enabled, the username is set to ``webhook``.
```

File: source/cloud/cloud-integrations/cloud-incoming-webhooks.rst [hunk 3]
```diff
@@ -70,24 +70,24 @@ This content will be displayed in the Town Square channel.
...
   
-  Similarly, `Enable integrations to override profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__ must be set to `true` in `config.json` to override profile picture icons. Enable them from **System Console > Integrations > Custom Integrations** in prior versions or **System Console > Integrations > Integration Management** in versions after 5.12 or ask your System Administrator to do so. If not enabled, the icon of the creator of the webhook URL is used to post messages.
+  Similarly, `Enable integrations to override profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__ must be set to ``true`` in ``config.json`` to override profile picture icons. Enable them from **System Console > Integrations > Integration Management** or ask your System Administrator to do so. If not enabled, the icon of the creator of the webhook URL is used to post messages.
```
Comment (justinegeffen) [inline source/cloud/cloud-integrations/cloud-incoming-webhooks.rst:75]:
```suggestion
  Similarly, `Enable integrations to override profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__ must be set to ``true`` in ``config.json`` to override profile picture icons. Enable them from **System Console > Integrations > Integration Management** or ask your System Admin to do so. If not enabled, the icon of the creator of the webhook URL is used to post messages.
```

File: source/cloud/cloud-integrations/cloud-outgoing-webhooks.rst [hunk 4]
```diff
@@ -100,15 +100,15 @@ which would render in Mattermost as:
 Messages with advanced formatting can be created by including an `attachment array <https://docs.mattermost.com/developer/message-attachments.html>`_ and `interactive message buttons <https://docs.mattermost.com/developer/interactive-messages.html>`_ in the JSON payload.
 
 .. note::
-  `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to `true` in `config.json` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your System Administrator. If not enabled, the username is set to `webhook`.
+  `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to ``true`` in ``config.json`` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your System Administrator to do so. If not enabled, the username is set to webhook.
```
Comment (justinegeffen) [inline source/cloud/cloud-integrations/cloud-outgoing-webhooks.rst:103]:
```suggestion
  `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`__ must be set to ``true`` in ``config.json`` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your System Admin to do so. If not enabled, the username is set to ``webhook``.
```

File: source/cloud/cloud-integrations/cloud-outgoing-webhooks.rst [hunk 5]
```diff
@@ -100,15 +100,15 @@ which would render in Mattermost as:
...
 
-  Similarly, `Enable integrations to override profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__ must be set to `true` in `config.json` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your System Administrator. If not enabled, the icon of the creator of the webhook URL is used to post messages.
+  Similarly, `Enable integrations to override profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__ must be set to ``true`` in ``config.json`` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your System Administrator to do so. If not enabled, the icon of the creator of the webhook URL is used to post messages.
```
Comment (justinegeffen) [inline source/cloud/cloud-integrations/cloud-outgoing-webhooks.rst:105]:
```suggestion
  Similarly, `Enable integrations to override profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__ must be set to ``true`` in ``config.json`` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your System Admin to do so. If not enabled, the icon of the creator of the webhook URL is used to post messages.
```

**Review by justinegeffen — COMMENTED** [review]
Thanks, @cwarnermm! Some minor feedback but otherwise this looks good. :)

**Review by justinegeffen — APPROVED** [review]

**Comment by cwarnermm** [comment]
@justinegeffen Thank you for the great editor feedback!

File: source/cloud/cloud-mobile/troubleshooting-mobile.rst (hunk 1 above)
Comment (adamjclarkson) [reply source/cloud/cloud-mobile/troubleshooting-mobile.rst:13]: Yeah agree https://customers.mattermost.com/cloud/contact-us is a nicer page than the zendesk link. Both reach the same team and are acceptable but the Contact Us page is a better experience.

Comment (adamjclarkson) [reply source/cloud/cloud-mobile/troubleshooting-mobile.rst:13]: And change the language to "If you're a Cloud subscriber, you may reach our Support team 
via our [Contact Us page](https://customers.mattermost.com/cloud/contact-us)

**Review by adamjclarkson — APPROVED** [review]
Approved with the suggested change to link to https://customers.mattermost.com/cloud/contact-us instead of zendesk.

File: source/cloud/cloud-mobile/troubleshooting-mobile.rst (hunk 1 above)
Comment (cwarnermm) [reply source/cloud/cloud-mobile/troubleshooting-mobile.rst]: Thanks, @adamjclarkson!

**Pull request merged**